server/routes/teams.ts

```diff
@@ -86,6 +88,7 @@ export default function routes({
         channelRecommendations,
         channelTree,
         hasLegacyChannels,
```

Comment:
I've get rid of the `const` and just use `hasLegacyChannels: legacyChannelCount > 0,`